src/SampleBlazorApplication/Program.cs

```diff
@@ -0,0 +1,54 @@
+using Ninject;
+using Ninject.Web.AspNetCore;
+using SampleBlazorApplication.Data;
+
+namespace SampleBlazorApplication;
+
+public class Program
+{
+	public static void Main(string[] args)
+	{
+		var kernel = CreateKernel();
+		var builder = WebApplication.CreateBuilder(args);
+
+		builder.Host.UseServiceProviderFactory(new NinjectServiceProviderFactory(kernel));
```

Comment:
This is almost the same as the default Blazor sample project with some very small modifications:
* I added the _class_ because I really don't like the "naked" Program.cs style
* We're injecting our own `NinjectServiceProviderFactory` right here
* We create a custom `AspNetCore` kernel with `CreateKernel`

Reply:
That makes it easy to use. Looks good for me.